### ISSUE TITLE
undefined: v1.Initializers

### ISSUE THREAD

**Issue by brandon-bethke-neudesic** [kubernetes/client-go#248]
I get the following compile error when I import "k8s.io/api/core/v1" into my file for the type v1.ConfigMap.

k8s.io/api/core/v1
../k8s.io/api/core/v1/types.go:213: undefined: v1.Initializers

I am using v3.0.0 of client-go as recommended in README.md
I am using to godep management tool and instructions as recommended in INSTALL.md

**Comment by ericchiang** [comment]
v3.0.0 doesn't use the `k8s.io/api` repo: https://github.com/kubernetes/client-go/blob/v3.0.0/kubernetes/typed/core/v1/configmap.go#L24

**Comment by brandon-bethke-neudesic** [comment]
Thank you and I apologize. The example code I saw must have been for a different version.

**Comment by fatsheep9146** [comment]
@brandon-bethke-neudesic 
I came across similar questions, too. I used "k8s.io/client-go/kubernetes" with "k8s.io/api/core/v1"(I wanted to use v1.Service), the compiler gave me same errors just like yours.

./k8s.io/api/core/v1/types.go:213: undefined: v1.Initializers

I solved this problem by using the "v1" inside the client-go, I changed my imported library like following,

import (
    "k8s.io/client-go/kubernetes"
    "k8s.io/client-go/pkg/api/v1"
)
The code is compiled successfully with no errors. Hopefully this would help you.
